api/src/index.rs

```diff
@@ -109,3 +109,7 @@ async fn handle_rejection(err: Rejection) -> Result<impl Reply, Infallible> {
     }
     Ok(reply::with_status(body, code))
 }
+
+fn open_api_html() -> String {
+    OPEN_API_HTML.replace("hideTryIt=\"true\"", "")
```

Comment:
remove `hideTryIt` flag when server renders the html, because `tryIt` defaults to send request to the same base url of the html file.
We set `hideTryIt` to true for other places like diem doc website for now, later when we have public node running, we can point the url to the public node and enable try it